.appveyor.yml

```diff
@@ -54,6 +54,12 @@ environment:
     - PYTHON: "C:\\Python38-x64"
       PYTHON_VERSION: "3.8"
       PYTHON_ARCH: "64"
+    - PYTHON: "C:\\Python38"
+      PYTHON_VERSION: "3.9"
+      PYTHON_ARCH: "32"
+    - PYTHON: "C:\\Python38-x64"
+      PYTHON_VERSION: "3.9"
+      PYTHON_ARCH: "64"
```

Comment:
It seems that appveyor have installed Python 3.9 only for their Visual Studio 2019 image. If we want to test with Python 3.9 we'll have to use Visual Studio 2019 image.

Reply:
Yes, I have updated the configuration.